Gruntfile.js

```diff
@@ -36,7 +36,9 @@ module.exports = function(grunt) {
             "vendor/underscore-1.8.3.js",
             "vendor/backbone-1.1.2.js",
             "vendor/jquery.dataTables-1.9.4.js",
-            "vendor/mock-ajax.js"
+            "vendor/mock-ajax.js",
+            "vendor/bootstrap-3.1.1-dist/js/bootstrap.js",
```

Comment:
The bootstrap is needed in the tests for the popoverMenu. I'm not sure what a better way would be. Any suggestions?